apps/cli/src/commands/shell.ts

```diff
@@ -24,8 +24,14 @@ export default class Shell extends BaseCommand<typeof Shell> {
             summary: "path to the configuration file",
         }),
         "run-as-root": Flags.boolean({
-            description: "run as root user",
             default: false,
+            description: "run as root user",
+            summary: "run the cartesi machine as the root user",
+        }),
+        shell: Flags.string({
+            default: "/bin/sh",
+            description: "shell command to run",
+            summary: "shell to run",
```

Comment:
```suggestion
        command: Flags.string({
            default: "/bin/sh",
            description: "command to run",
            summary: "command to run",
            char: "C",
```

Won't it be strang `cartesi shell --shell=/bin/bash`  ?

I think that a `--command` would be better, and it's like `flyctl` does with `flyctl ssh console -C | --command`.